<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleFour</class>
 <widget class="QDialog" name="ExampleFour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Приклад 4</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="7">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4;&quot;&gt;{x:0&amp;lt;x&amp;lt;p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4;&quot;&gt;-p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#3465a4;&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;{x:p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;&amp;lt;x&amp;lt;p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;+p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;-p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#73d216;&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400; vertical-align:sub;&quot;&gt;1 &lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;U A&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;{x:p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;-p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;&amp;lt;x&amp;lt;p&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-style:italic; color:#edd400;&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Багатократне моделювання</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Моделювати</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Одиничне моделювання</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Моделювати</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>p1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>p2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>p3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_3">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;P(A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt; U A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;) = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_3">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;P(A&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;2 &lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;\ A&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;) = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;P(A&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;1 &lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;\ A&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;) = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
